portfolio/src/main/webapp/css/comments-style.css

```diff
@@ -98,14 +111,14 @@ h3 {
 
 #form-container {
   padding-bottom: 25px;
-  width: 60%;
+  width: 100%;
 }
 
 form {
   width: 100%;
 }
 
-/* Social media icons styling */
+/** Social media icons styling */
```

Comment:
These should not be JSDoc comments right? Or should they be?

Reply:
Ah, in a previous pull request JSDoc change was suggested but I checked the CSS style guide and they should be `/* */`.

Reverted

Reply:
Thanks for checking!